server/src/main/java/org/solarbank/server/dto/CalculateRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.solarbank.server.dto;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+import org.solarbank.server.dto.CalculateResult;
+
+@Data
+@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+public class CalculateRequest {
+    @Valid
+    @NotNull(message = "Location must be provided")
```

Comment:
I thought I should use enum for these strings, `ValidationMessage.LOCATION_NULL.getMessage() `but can't make method calls inside the annotation. I then tried to have a class like the below, but still wouldn't work. Couldn't figure out a workaround?
```
public class ValidationMessages {
    public static final String LONGITUDE_NULL = ValidationMessage.LONGITUDE_NULL.getMessage();
}
```